lib/npg_tracking/Schema/Result/Instrument.pm

```diff
@@ -159,6 +165,8 @@ __PACKAGE__->add_columns(
   },
   "percent_complete",
   { data_type => "tinyint", extra => { unsigned => 1 }, is_nullable => 1 },
+  "lab",
+  { data_type => "varchar", is_nullable => 1, size => 255 },
```

Comment:
Not in agreement schema.txt below. I assume this is the correct one.

Reply:
`schema.txt` was the correct one. I regenerated this file so they should match now.